index.html

```diff
@@ -14,21 +14,115 @@
+          <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
+          <li><img src="./images/Vector.jpg" alt="email"></li>
+          <li><img src="./images/Vector-2.jpg" alt="github"></li>
+          <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
+        </ul>
+      </section>
+
+      <section class="previous-work">
+        <section class="project">
+          <img src="./images/Snapshoot-Portfolio.png" alt="project 1 snapshot">
+          <h1>Tonic</h1>
+          <ul class="tech">
+            <li class="company-name"><a>company</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>Back End Dev</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>2015</a></li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="prog-languages">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <button type='button'>See Project</button>
+        </section>
+
+        <section class="project">
+          <img src="./images/Snapshoot-Portfolio-3.png" alt="snapshot1">
+          <h1>Tonic</h1>
+          <ul class="tech">
+            <li class="company-name"><a>company</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>Back End Dev</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>2015</a></li>
+          </ul>
+          <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+          <ul class="prog-languages">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <button type='button'>See Project</button>
+        </section>
+
+        <section class="project">
+          <img src="./images/Snapshoot-Portfolio4.png" alt="snapshot1">
+          <h1>multi-post stories</h1>
+          <ul class="tech">
+            <li class="company-name"><a>company</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>Back End Dev</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>2015</a></li>
+          </ul>
+          <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+          <ul class="prog-languages">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <button type='button'>See Project</button>
+        </section>
+      </section>
+
+      <section id="about-me">
+        <h1>About Myself</h1>
```

Comment:
- [ ] [REQUIRED] Please, always use the `<h1>` tag only 1️⃣ once in the entire document 📄. Since [it represents the title of the document](https://html.com/tags/heading/).
- [ ] [OPTIONAL] Make sure that the words `"About"` and `"Myself"` in to separate lines. To match the Figma Design 🖼️.